projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -84,55 +97,195 @@ export class IgxTimePickerComponent implements ControlValueAccessor, OnInit, OnD
         }
     }
 
+    /**
+     * An accessor that returns the value of the `_value` property.
+     * ```html
+     *@ViewChild("MyPick")
+     *public pick: IgxTimePickerComponent;
+     *ngAfterViewInit(){
+     *    let pickSelect = this.pick.value;
+     * }
+     * ```
+     */
     get value(): Date {
         return this._value;
     }
 
+    /**
+     * An @Input property that allows you to disable the component. By default it's not applied.
+     * ```html
+     * <igx-time-picker [isDisabled]="true" [vertical]="true" format="h:mm tt" ></igx-time-picker>
+     * ```
+     */
     @Input()
     public isDisabled = false;
 
+    /**
+     * An @Input property that renders OK button with custom text, which commits the selected time. By default `okButtonLabel` is set to OK.
```

Comment:
@alexalex972 ",which commits the selected time" is not needed. The property is about the button text.